File: mdc/DataCenter/MDC/Data/Dtos/MapPointDto.cs
using System;

namespace MDC.Data.Dtos
{
    public class MapPointDto
    {
        public string Name { get; set; }

        public int GroupId { get; set; }

        public string World { get; set; }

        public double X { get; set; }

        public double Y { get; set; }

        public double Z { get; set; }

        public DateTime CreatedDate { get; set; }
    }
}

File: server/plugins/MineParkCore/src/minepark/mdc/dtos/MapPointDto.php
<?php
namespace minepark\mdc\dtos;

class MapPointDto extends BaseDto
{
    public string $name;

    public string $level;

    public float $x;

    public float $y;

    public float $z;

    public int $groupId;

    public string $createdDate;
}
?>